### ISSUE TITLE
Content jumps in case studies list

### ISSUE THREAD

**Issue by Sm1lEE** [21stdigital/21st.digital#401]
# Problem

Einige Livewire-Komponenten, z. B. die Case Studies Liste auf der [Startseite](https://21st.digital/) oder der [Case Studies Übersicht Seite](https://21st.digital/case-studies), verursachen ein Springen des Layouts. Das erschwert das Erfassen des Website-Inhaltes und sorgt für ein unrundes Benutzererlebnis.

# User Story

Als Besucher der Website möchte ich die Inhalte so schnell wie möglich präsentiert bekommen, ohne dass diese erst später nachgeladen werden und durch Verschieben der anderen Website-Inhalte für ein unruhiges Erlebnis sorgen.

# Vorhaben

- Eliminierung der Layout-Sprünge
- Evaluieren, ob eine Art SSR in den Livewire Komponenten verwendet werden kann, damit der Inhalt initial bereits vorhanden ist und von Livewire nur bei Bedarf aktualisiert wird

# Abgrenzung

# Akzeptanzkriterien

- Bei der Darstellung der Case Studies Liste findet kein nachträglicher Layoutsprung mehr statt
- Der User sieht den Inhalt der Case Studies Liste und den restlichen Inhalt der Webseite zur gleichen Zeit
- Es wird trotzdem das durch Livewire angestrebte Auto-Update der Inhalte beibehalten

## Beispiele

https://github.com/user-attachments/assets/d020a78f-dbde-4bfe-9fba-ff9823509c87

https://github.com/user-attachments/assets/1ceb4c5a-bfe2-4a02-aac5-54221ea5dd17

**Comment by robocopklaus** [comment]
@Sm1lEE ja, die Komponente sollte aus meiner sicht nicht clientseitig gerendert werden. livewire brauchen wir ja für unsere website gar nicht.

**Comment by Sm1lEE** [comment]
Ich glaube, @JimSeven wollte da testen, ob man eine Art SWR hinbekommt: Initial sollen die Teaser-Items aus dem Cache vom Server kommen und dadurch schnell ausgeliefert werden. Gleichzeitig wird per Livewire ein Request gemacht und geschaut, ob es Änderungen gibt. Falls ja, werden die Inhalte dann ausgetauscht. Das fand ich ganz gut.

Ich habe vorhin schon probiert, ob man nicht schon direkt die initialen Daten anzeigen kann. Es war aber immer so, dass er nichts initial gezeigt hat und dadurch der Layout-Sprung kommt. Vielleicht hab ich es auch noch nicht gefunden, aber idealerweise müssten Inhalte auch angezeigt werden, wenn JS deaktiviert ist. So hatte ich das auch getestet. Da war leider nichts im Teaser zu sehen.

**Comment by robocopklaus** [comment]
ah ok. für mich wäre das aber nicht schlimm, wenn wir "veraltete" daten zeigen. so häufig ändern wir ja die website nicht.

**Comment by Sm1lEE** [comment]
Das stimmt wohl. Eine schöne Übung wäre es aber schon, die Anforderung hatten wir bei einigen Projekten schon öfter. Wir sollten dann nur mal festlegen, wie weit wir das verfolgen wollen für die Live-Version oder ob wir bis zur Lösung eine andere "besser funktionierende" Variante verwenden.
